src/components/backup/BackupConfirmPasswordStep.js

```diff
@@ -231,7 +230,7 @@ const BackupConfirmPasswordStep = ({ setAppearListener }) => {
       }
     } catch (e) {
       logger.log('Error while backing up', e);
-      passwordRef.current?.focus();
+      setTimeout(passwordRef.current.focus());
```

Comment:
did this help? (just curious)

Reply:
don't think so

didn't mean to include this, just removed